src/components/DirectorySettingsModal/DirectorySettingsSchema.jsx

```diff
@@ -23,12 +27,20 @@ export const DirectorySettingsSchema = (existingTitlesArray = []) =>
         "Title is already in use. Please choose a different title."
       )
       .when("$type", (type, schema) => {
+        if (type === "mediaDirectoryName")
+          return schema
+            .transform((value) => deslugifyDirectory(value))
+            .test(
+              "Special characters found",
+              'Title cannot contain any of the following special characters: ~%^*+./`;{}[]"<>',
```

Comment:
Chatted offline about this - `#` cannot be used in image filenames, however it can be used for the media directory names. I used the wrong validation above, and have rectified it to allow for `#` in media directory names. 

However, while looking into this above, I realised that our frontend and backend was not correctly supporting encoded media directory names, so I've put in the following commits to rectify this:

- https://github.com/isomerpages/isomercms-frontend/pull/716/commits/b4efdcc91f2f4b18101240ed01765b64204bb1f6 (update FolderCard link to link to encoded mediaDirectoryName)
- https://github.com/isomerpages/isomercms-frontend/pull/719/commits/688bbb5bc1c65c1cb09f5d7d49b01eedbd019811 (remove validation against `#` in media directory names)
- https://github.com/isomerpages/isomercms-backend/pull/324/commits/c54204402d694250a68e25b4fc96db43065842cf (encode mediaDirectoryNames when returning mediaUrl)

Also put in commits to reduce test flakiness for timeouts:
- https://github.com/isomerpages/isomercms-frontend/pull/717/commits/f9629baab728d791dd932e17a6c1bb5e5162e683

Reply:
The above commits (apart from the backend changes) are in [media/refactor-temp](media/refactor-temp) as well for ease of testing.